docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  web:
+    #For x64 architecture
+    #image: kodomoheichou/moodle-web:x64
+    image: kodomoheichou/moodle-web:arm
+    container_name: moodle-web
+    ports:
+      - "80:80"
+    depends_on:
+      - db
+    volumes:
+      - ./custom.ini:/usr/local/etc/php/conf.d/custom.ini
+      #Remove this line for production testing
+      - ./server/moodle/mod/livequiz:/var/www/html/mod/livequiz
+
+  db:
+    image: postgres:16
+    container_name: moodle-postgres-db
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - ./initdb.sql:/docker-entrypoint-initdb.d/initdb.sql
+    environment:
```

Comment:
If possible, I'd prefer if we stick with the same database provided by Ida (mariadb or mysql).

Reply:
If possible, I'd prefer we did not. Men vi kan lige snakke om det, ellers bliver det på dagsordenen til næste PO møde 😄  Der er mange fordele, flere stod også på dagsordenen for dagens PO møde 😸

Reply:
I know Postgres is the better database system. But Ida mentioned today during the PO meeting today that everything needs to get merged by her, so it should follow the general systems she has outlined herself.

Reply:
Jeg vil helst tage diskussionen ansigt til ansigt. 😄  For jeg tror måske ida tror at hvis vi bruger postgres til udvikling kan hun ikke længere bruge HeidiSql, men dette er en forkert antagelse. Jeg er åben for yderligere diskussion, men dette skal helst forgå gennem min PO Christoffer, eller ansigt til ansigt 😄

Reply:
Understood.

This pr is stalled until the next PO meeting with Ida.

Reply:
Eller indtil yderligere diskussion 👯